### ISSUE TITLE
Typo3 v12?

### ISSUE THREAD

**Issue by agriaIT** [AgenturPottkinder/typo3_forum#370]
Hi Pottkinder,

are there any plans for Typo3 v12 in the near future?

Thanks & best regards

**Comment by Avalarion** [comment]
Yes there are. We are working on this since 01.12 very actively. But is it is a lot of work.

**Comment by agriaIT** [comment]
Is there a chance to get a running version at the end of april?

**Comment by Avalarion** [comment]
Realistic timeframe. We have a running version with bugs right now but it is part of a current project we maintain.

**Comment by agriaIT** [comment]
Very good. If you provide a version for testing that works as far as possible, we will be happy to test it as soon as possible.

**Comment by boosim** [comment]
Hi @Avalarion,

are there any news about your plans for the TYPO3 12 version?
Is April still a possible timeframe?

Thanks & kind regards

**Comment by xnet74** [comment]
Hello,

there are any news about your plans for the TYPO3 12 version?

Thanks Gero

**Comment by Avalarion** [comment]
Hey there,

so we have a running, but still in test, version for TYPO3 v12 and will try to add it this month to github, please give us a moment.

Welcome to 2025,

Bastian

**Comment by agriaIT** [comment]
Hey there,

so we have a running, but still in test, version for TYPO3 v12 and will try to add it this month to github, please give us a moment.

Welcome to 2025,

Bastian

**Comment by stephangrass** [comment]
Hey. And for TYPO3 13?

**Comment by Avalarion** [comment]
Not yet started as we have no TYPO3 13 with a forum yet. This is the goal AFTER providing V12 code here^^.

**Comment by agriaIT** [comment]
Hey. And for TYPO3 13?

**Comment by agriaIT** [comment]
Not yet started as we have no TYPO3 13 with a forum yet. This is the goal AFTER providing V12 code here^^.

**Comment by agriaIT** [comment]
Hi Pottkinder, hi @Avalarion , do you have any news for the release date for V12?

**Comment by agriaIT** [comment]
Thank you very much for updating to V12 on Github and Packagist :-)
